src/components/SliderInput.tsx

```diff
@@ -66,6 +66,7 @@ export const SliderInput = ({ min, max, prefixLabel, suffixLabel, name, step = 1
             if (error) {
               setError(name, { message: error });
             } else {
+              clearErrors(name);
```

Comment:
Forgot I added this, it should fix https://github.com/grafana/synthetic-monitoring-app/issues/378